docs/api/v3.rst

```diff
@@ -412,6 +412,10 @@ Versions listing
 
     :query boolean active: return only active versions
     :query boolean built: return only built versions
+    :query string privacy_level: return versions with specific privacy level
+    :query string slug: return versions with matching slug
+    :query boolean uploaded: return only uploaded versions
```

Comment:
Uploaded shouldn't be here, or at least it shouldn't be documented. This is a very old feature we don't expose anymore. And I'm not really sure we have versions marked as uploaded.

Reply:
Okay, sure! I did not know that. Removed it from documentation but not sure if I should remove it from filters as it was already there.

Reply:
I think it was added in the filters to make use of it internally only